combinetf2/physicsmodels.py

```diff
@@ -0,0 +1,399 @@
+import re
+
+import tensorflow as tf
+
+
+class PhysicsModel:
+    """
+    A class to output histograms without any transformation.
+    """
+
+    def __init__(self, indata):
+        self.name = "basemodel"  # Used for the output dictionary
+        # channel info of the resulting histogram, needed for the writing
+        self.channel_info = indata.channel_info
+        # if data histograms are stored or not, and if chi2 is calculated
+        self.has_data = True
+        # a list of strings, the result will be written under these keys
+        self.identifiers = []
+
+    def make_fun(self, fun_flat, inclusive=True):
+        return fun_flat
+
+    def get_data(self, data):
+        fun = self.make_fun(lambda: data)
+        return fun(), None
```

Comment:
Couldn't this just directly be ```return data, None``` ?

Reply:
In this case yes but if a more complex PhsicsModel is implemented then this should gives the right values for data without the need to override the function.